docs/install/windows.md

```diff
@@ -1,37 +1,405 @@
+
+```C
+     if (nrnmpi_myid < 1)
+        if (nrn_nobanner_ == 0) {
+            Fprintf(stderr, "%s\n", nrn_version(1));
+            printf("\nNow It is my NEURON version!!!! \n\n");
+            Fprintf(stderr, "%s\n", banner);
+            IGNORE(fflush(stderr));
+        }
+```
+and save.
+
+#### Step 14.C: Compiling the new code.
+
+Compile this new code. We can compile using Step 15. (Do it and come back here) 
+
+#### Step 14.D: Replacing the `libnrniv.dll` file
+
+It will create a new `libnrniv.dll` (and `libnrnmpi_msmpi.dll` and a new `libnrnpython3.XX.dll` and new `nrniv`, but just the first one matters for THIS example) at `nrn\build\bin` (at `C:\Users\User\Neuron\nrn\build\bin` or at `C:\msys64\home\User\Neuron\nrn\build\bin` if you did the installation process here). 
+
+We can replace the old file `libnrniv.dll` at `C:\nrn\bin` for this new one and voila! We get our change implemented!! Let's see
+
+#### Step 14.E: Checking the new `nrngui` 
+
+Run `nrngui` (inside the folder `C:\Users\User\Desktop\NEURON 9.0.0 AMD64` double click in `nrngui`)
+
 ```
-This will download all required installers and corresponding versions(`Python`, `MPI`, `nsis`)
+NEURON -- VERSION X.X.X 2024-07-11
+
+Now it is my NEURON version!!!!
+
+Duke, Yale, and the BlueBrain Project -- Copyright 1984-2022
+See http://neuron.yale.edu/neuron/credits
+
+oc>
+```
+
+------------------------------------------------------------ 
+
+### Step 15: Compile new NEURON code
+
+
+PS: Useful commands:  
 
-Then launch the installation script:
 ```powershell
-PS C:\workspace\nrn> .\ci\win_install_deps.cmd
+cd /c/Users/User/Neuron/nrn/build
+``` 
+to reach files outside of `C:\msys64\`.
+
+```
+echo $PATH
+which gcc 
+which cmake 
+```
+To compile new NEURON code, you can (always from a `mingw64` terminal): 
+
+#### Step 15.A: run win_build_cmake.sh 
+
+From nrn file where the source code is (example: `C:\Users\User\Neuron\nrn\ci` or `C:\msys64\home\User\Neuron\nrn\ci` if you copy a version here) run: 
+```
+.\win_build_cmake.sh
+```
+or 
+ 
+#### Step 15.B: Run "cmake .." 
+
+From nrn file where is the source code (example: `C:\Users\User\Neuron\nrn\build` or `C:\msys64\home\User\Neuron\nrn` if you copied a version here - see Step 18) run: 
+
+```
+cmake .. -GNinja
+```
+
+or 
+
+#### Step 15.C: ninja
+
+Run `ninja` from the build file (example: `C:\Users\User\Neuron\nrn\build` or `C:\msys64\home\User\Neuron\nrn\build` if you copied a version here) 
+```
+ninja
+``` 
+
+### Step 16: Dependencies
+
+Python will not have all dependencies, so you can install them doing:  
+
+```
+python -m pip install plotly 
+python -m pip install matplotlib 
+python -m pip install pandas 
+``` 
+
+
+## Creating a new environment for NEURON development 
+It my be useful maintain the original version of NEURON installed (at `C:\home\User\Neuron\nrn` for example) and have a new version that you can do your changes in code. We will set up this new environment and version.
```

Comment:
It **may** be useful **to** maintain....

(typo)

... that you can use for development.